src/signup.js

```diff
@@ -57,44 +67,53 @@ function matchFailPswdCheck(e) {
   }
 }
 
-function seeOrNotPassword(e) {
+function togglePassword(e) {
   if (pswdInput.type === 'password') {
     pswdInput.type = 'text';
-    e.target.src = '/images/eye-on.svg';
+    e.target.src = EYE_ON_PATH;
   } else if (pswdInput.type === 'text') {
     pswdInput.type = 'password';
-    e.target.src = '/images/eye-off.svg';
+    e.target.src = EYE_OFF_PATH;
   }
 }
 
-function seeOrNotPasswordCheck(e) {
+function togglePasswordCheck(e) {
   if (pswdCheck.type === 'password') {
     pswdCheck.type = 'text';
-    e.target.src = '/images/eye-on.svg';
-  } else if (pswdCheck.type === 'text') {
+    e.target.src = EYE_ON_PATH;
+  } else if (pswdInput.type === 'text') {
     pswdCheck.type = 'password';
-    e.target.src = '/images/eye-off.svg';
+    e.target.src = EYE_OFF_PATH;
   }
 }
 
-let checkNoError = function () {
-  let emailError = emailInput.nextElementSibling.className !== 'alert';
-  let pswdError = pswdInput.nextElementSibling.className !== 'alert';
-  let pswdCheckError = pswdCheck.nextElementSibling.className !== 'alert';
+const checkNoError = function () {
+  const emailError = emailInput.nextElementSibling.className !== 'alert';
+  const pswdError = pswdInput.nextElementSibling.className !== 'alert';
+  const pswdCheckError = pswdCheck.nextElementSibling.className !== 'alert';
   return emailError && pswdError && pswdCheckError;
 };
```

Comment:
각 엘리먼트에 대해 같은 검사를 수행학 있어서 `every` 메서드로 간소화 할 수 있을 것 같아요

- 추천 키워드: `every` 메서드, `some` 메서드

```suggestion
const checkNoError = () => {
  const elements = [emailInput, pswdInput, pswdCheck];
  return elements.every((element) => element.nextElementSibling.className !== 'alert');
};
```